tests/__fixtures__/utils/location-operations.ts

```diff
@@ -11,12 +11,12 @@ let userLocation: LatLong = {
 };
 
 export async function updateLocationIfNeeded(
-  answersActions: AnswersActions,
+  SearchActions: SearchActions,
```

Comment:
there are some more places in the tests where the variable name capitalization needs to be fixed